pytorch_lightning/utilities/argparse.py

```diff
@@ -135,37 +134,84 @@ def get_init_arguments_and_types(cls) -> List[Tuple[str, Tuple, Any]]:
     return name_type_default
 
 
-def add_argparse_args(cls, parent_parser: ArgumentParser) -> ArgumentParser:
-    r"""Extends existing argparse by default `Trainer` attributes.
+def get_abbrev_qualified_cls_name(cls):
+    assert isinstance(cls, type), repr(cls)
+    if cls.__module__.startswith("pytorch_lightning."):
+        # Abbreviate.
+        return f"pl.{cls.__name__}"
+    else:
+        # Fully qualified.
+        return f"{cls.__module__}.{cls.__qualname__}"
+
+
+def add_argparse_args(
```

Comment:
did you only move this or did you make changes too?

Reply:
Didn't mean to move (silly git diff!), but had inserted the `get_abbrev_qualified_cls_name` function above, and added `add_argument_group` option.